src/main/scala/nl/biopet/tools/extracttagsfromgtf/ArgsParser.scala

```diff
@@ -27,11 +27,11 @@ import nl.biopet.utils.tool.{AbstractOptParser, ToolCommand}
 
 class ArgsParser(toolCommand: ToolCommand[Args])
     extends AbstractOptParser[Args](toolCommand) {
-  opt[File]('i', "refflat") required () valueName "<file>" action { (x, c) =>
-    c.copy(outputFile = x)
-  } text "Input refFlat file. Mandatory"
-  opt[File]('o', "gtfFile") required () valueName "<file>" action { (x, c) =>
+  opt[File]('i', "gtfFile") required () valueName "<file>" action { (x, c) =>
     c.copy(gtfFile = x)
+  } text "Input refFlat file. Mandatory"
```

Comment:
Shouldn't this be: "Input gtf file. Mandatory"?

Reply:
Yes indeed, fixed now